my-react-app/src/app/screens/Login/index.js

```diff
@@ -19,4 +21,9 @@ class LoginFormContainer extends Component {
   }
 }
 
-export default LoginFormContainer;
+const mapDispatchToProps = dispatch => ({
+  login: credentials =>
+    dispatch(actionCreators.login({ email: credentials.email, password: credentials.password }))
+});
+
+export default connect(mapDispatchToProps)(LoginFormContainer);
```

Comment:
The first parameter of connect is `mapStateToProps`. If you don't have that , do 
```
export default connect(null,mapDispatchToProps)(LoginFormContainer);
```